core/src/main/java/com/google/zxing/common/StringUtils.java

```diff
@@ -33,6 +33,7 @@
   private static final Charset PLATFORM_DEFAULT_ENCODING = Charset.defaultCharset();
   public static final Charset SHIFT_JIS_CHARSET = Charset.forName("SJIS");
   public static final Charset GB2312_CHARSET = Charset.forName("GB2312");
+  public static final Charset ISO_8859_1 = Charset.forName("ISO-8859-1");
```

Comment:
I think there is a symbol in StandardCharsets for this already

Reply:
>I think there is a symbol in StandardCharsets for this already
Absolutely. I changed the code to make use of it.

>Don't we already have a flag to force the charset, or does that not help here?
I didn't verify but it would be only of limited help since spec conform input can be multi ECI so that a code can for example contain two or more KANJI or BYTE segments, each one with a different encoding.